core/src/epicli/data/common/ansible/playbooks/roles/postgresql/tasks/upgrade/upgrade.yml

```diff
@@ -35,6 +35,27 @@
   set_fact:
     pg_primary_node: "{{ pg_primary_node_found.stdout | default('') }}"
 
+- name: PostgreSQL | Query custom tablespaces
```

Comment:
How about using `postgresql_info` module?
Please check https://docs.ansible.com/ansible/latest/collections/community/postgresql/postgresql_info_module.html#return-tablespaces

Reply:
It was my thought to do so initially, but this module doesn't display tablespaces' filesystem location and it should be requested somehow.

Reply:
Oh, didn't know, sorry then :flushed:
You can consider adding  comment about that.